### ISSUE TITLE
Aragon Nest Proposal - Decentralized Cryptocurrency Exchange Traded Fund Token

### ISSUE THREAD

**Issue by jtiannelli** [aragon/nest#16]
ETF Token
Cryptocurrency Index backed Token

## Abstract
ETF Token aims to create an actively managed fund of cryptocurrencies – a token representing the best available assets available throughout the entire market.  ETF token will be fully backed by verifiable blockchain smart contract transactions which will automatically add to the index of currencies the exact value of each buy order. ETF token will provide reduced volatility, suitable for: long term investment in blockchain based cryptocurrencies, short term fund transfers for day traders, and investors looking for a safer alternative currency to hold when not actively trading. It is meant to appeal to the smallest potential investor in crypto currency to the largest investors and original innovators aiming to reduce their exposure to daily volatility. 

## Deliverables
1. An Ethereum based token which will be 100% backed by an actively managed leading index of cryptocurrencies.
2. A set of AragonOS-compatible smart contracts to transfer ethereum from token purchases into the backing index of cryptocurrencies.  Making backing the investments mandatory as opposed to questionable like with Tether them with multiple tokens, with the contracts burning them afterwards
3. A web interface to let users purchase Tokens, exercise their voting rights and otherwise interact with the ETF Foundation and community.

## Grant size

Funding: Up to $50k in ETH, split into chunks paid out over achieved deliverables.

Success reward: The continued support and friendship of this community!

## Application requirements
Concept:
•	Initial tokens and all tokens issued in perpetuity would be 100% backed by other coins (or other investments to be determined by the coin holder’s votes).  
•	The ETF foundation will issue coins as demand increases and use the proceeds to increase the size of the backing fund, so that each ETF Coin would be backed by ETF foundation’s portfolio of crypto investments.  
•	Downloadable wallets and purchase capabilities would be available on the website. 
•	Internal transaction processors would receive a fractional percentage of the coin for processing/verifying these transactions.  These coins would not be created by mining but rather “paid” to the foundation via a small fee associated with each buy and sell transaction made through the ETF web application, essentially a less expensive system than, but similar to, the way many other types of managed funds operate. 

Whitepaper: (Draft v.1.0.1)
https://www.etftoken.org/whitepaper

## Development timeline

The development timeline will be the following one in regards to each deliverable:

1. Feb 2018 - Finalized Whitepaper and Plan of Action
2. March 2018 - Working Concept waiting for implementation Funding
3. May 2018 - ICO launch (fully functioning platform)

Testnet launch will be expected during March 2018, with mainnet launch being late april/ early may 
[ETF Token White Paper V1.0.1.pdf](https://github.com/aragon/nest/files/1693395/ETF.Token.White.Paper.V1.0.1.pdf)
2018

**Comment by jtiannelli** [comment]
[ETF Token Application for Review.pdf](https://github.com/aragon/nest/files/1693400/ETF.Token.Application.for.Review.pdf)

**Comment by luisivan** [comment]
Thanks for opening the issue, although this doesn't really follow the guideline for applications (since there's not a proposal for this grant, see https://github.com/aragon/nest/blob/master/Guide_for_submitting_a_request_for_funding.md) so I'll have to label it as rejected until the proper process is completed. I also have a question: if you plan to ICO, why the need for a grant?

**Comment by jtiannelli** [comment]
Wow! I am honored to get a response from you. My initial goal was to seek
no grant funding, but we realize we will have to advertise our token, we
were unprepared for the costs, considering this is a non-profit
organization.

The reason we need grant funding is that when someone contributes to this
ICO, the foundation does not get access to that money, the purchase is only
executed once the smart contract verifies the investment is deposited into
the backing index of crypto currencies.  Usually during an ICO the team
retains a high value of tokens for themselves to spend. However, this token
is supposed to be owned and operated by the community, we will not be
keeping ICO funds so will still need to fundraise to keep the token fund
100% backed by the cryptocurrencies that make up the index.  Essentially we
can't spend any of the money from the ICO because the community controls it.

I accept your rejection and will have the technical part of the team help
established better follow the proposal process. We are currently struggling
with some of these details because we are early in the process and are
making some of those final decisions.  We are seeking advisors to help us
make those decisions if you are interested.

After that, even if this token is not perfect for the Aragon Nest program,
I appreciate what you guys are doing and we will certainly be investing in
Aragon because of your core principle once the fund is established.

Thanks again,
Joe Iannelli

**Comment by luisivan** [comment]
Got it. Thanks for the kind words! @jtiannelli

**Comment by jtiannelli** [comment]
Even though this is currently closed. And ETF Token Foundation is not yet
ready for Aragon Nest.  We are still moving forward and are updating the
draft whitepaper daily.  This is the link for where we post the latest
updates. A lot has improved in the last two weeks.
https://www.etftoken.org/whitepaper

I think we share the same principles of seeing decentralized self governing
tokens as the ideal future for blockchain organization.  And hope to
continue to stay in touch with your organization as we develop ours.

Best regards,
JT

**Comment by jtiannelli** [comment]
Luis,

I took the last two weeks to make substantial concept and whitepaper improvements and also to get a little further with establishing more of our decentralized team and improving our roadmap.  I have updated the pull request to match the instructions for proposal submittal as you requested.  Happy to initiate a dialogue with you guys regarding this proposal, I am very passionate about this mission.  I think it is a necessary token that people will enjoy holding, and it will in turn support dozens of other world changing blockchain projects and decentralized exchanges that otherwise may not have achieved funding for multi-year successful development. 

Thanks for your consideration!
-JT
